src/test/java/duke/command/handler/CommandDeadlineHandlerTest.java

```diff
@@ -0,0 +1,52 @@
+package duke.command.handler;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import duke.command.CommandException;
+
+class CommandDeadlineHandlerTest {
+
+    @Test
+    void deadlineHandler_normalInputsValidation_success() {
+        List<String> normalInputs = Arrays.asList(
+            "deadline task-1 /by 2022-08-24",
+            "deadline task-2 /by 2022-08-24 23:59"
+        );
+        for (String normalInput : normalInputs) {
+            try {
+                new CommandDeadlineHandler(normalInput);
+            } catch (CommandException ignored) {
```

Comment:
Perhaps you can name the exception as "error" instead? I feel it would make more sense to use a noun.